handling-null-values-in-avro-and-protobuf/kafka/build.gradle

```diff
@@ -0,0 +1,146 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            url = uri("https://packages.confluent.io/maven/")
+        }
+        maven {
+            url = uri("https://plugins.gradle.org/m2/")
+        }
+        maven {
+            url = uri("https://jitpack.io")
+        }
+    }
+}
+
+plugins {
+    id 'java'
+    id 'idea'
+    id 'eclipse'
+    id "com.google.protobuf" version "0.9.4"
+    id "com.github.imflog.kafka-schema-registry-gradle-plugin" version "1.13.0"
+    id "com.github.davidmc24.gradle.plugin.avro" version "1.3.0"
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://packages.confluent.io/maven/")
+    }
+
+    maven {
+        url = uri("https://jitpack.io")
+    }
+}
+
+dependencies {
+    implementation 'com.google.protobuf:protobuf-java:3.21.1'
+    implementation 'org.apache.avro:avro:1.11.0'
+    implementation "io.confluent:kafka-avro-serializer:7.1.1"
+    implementation "io.confluent:kafka-protobuf-serializer:7.1.1"
+    implementation "io.confluent:kafka-protobuf-provider:7.1.1"
+
+    implementation 'org.apache.logging.log4j:log4j-api:2.18.0'
+    implementation 'org.apache.logging.log4j:log4j-core:2.18.0'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.18.0'
+
+    implementation('org.apache.kafka:kafka-streams:3.2.1') {
+        exclude group: 'org.apache.kafka', module: 'kafka-clients'
+    }
+    implementation('org.apache.kafka:kafka-clients:3.2.1')
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
suggest more recent dependency versions so that this is fresher... maybe `7.5.1` for confluent dependencies and `3.6.1` for AK